feat: ctrl click on error filename sends to the file line

prevostc opened the pull request:
When an error message is shown on an IDE console, most IDEs will allow
you to ctrl+click the file name to go directly to the proper file, line
and character. In order for this to work properly, the error filename,
line and character must have a specific format: "<filename>:<line
number>:<character number>".

johnnyreilly commented:
This is already supported by the default formatter but the syntax is different; it's like this:

`file.ts(2,6)`

This is a generally supported format for clickable errors. Certainly works in vs code etc. What are you using?

prevostc commented:
Oh nice this syntax works too \o/ Hold my beer.

johnnyreilly commented:
Yeah 😀

There's some relevant discussion here: https://github.com/facebook/create-react-app/pull/5624#issuecomment-442418402

prevostc committed:
feat: apply clickable filename format to codeframeFormatter

When an error message is shown on an IDE console, most IDEs will allow
you to ctrl+click the file name to go directly to the proper file, line
and character. In order for this to work properly, the error filename,
line and character must have a specific format: "filename.js(line,col)".
The codeFormatter already follows this format, this commit applies this
format to the codeframeFormatter.

prevostc commented:
@johnnyreilly I just pushed force a new commit to use this format, thanks for the link btw, very interesting.

johnnyreilly commented:
So this fixes up - the other formatter; makes it consistent with the default one?

prevostc commented:
@johnnyreilly Exactly. I came here from a frustrating next.js experience :)

johnnyreilly reviewed (approved):
Looks good to me!

piotr-oles commented:
:tada: This PR is included in version 1.5.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin)
- [GitHub release](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases/tag/v1.5.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
